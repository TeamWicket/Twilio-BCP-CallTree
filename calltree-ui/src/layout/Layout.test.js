import Layout from './Layout';
import React from 'react';
import { shallow } from 'enzyme';
import { Admin } from 'react-admin';

describe("Layout", () => {
    it("should match snapshot of Layout Component", () => {
        expect(shallow(<Admin><Layout /></Admin>)).toMatchSnapshot();
    })
    it("should return something", () => {
        expect(shallow(<Admin><Layout /></Admin>).length).toEqual(1);
    })
})
