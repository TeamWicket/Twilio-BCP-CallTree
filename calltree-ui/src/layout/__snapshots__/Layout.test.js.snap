// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should match snapshot of Layout Component 1`] = `ShallowWrapper {}`;
